Ticket- 51950 On our Load Tags, we changed all the different shipping addresses to "Broker"

sewa67 opened the pull request:
Please review the change and merge into development branch.
File - oerep/LoadtagProcs.p, oe/rep/lodxprnt9.i

sewa67 committed:
Ticket- 51950 On our Load Tags, we changed all the different shipping addresses to "Broker"
File - oerep/LoadtagProcs.p, oe/rep/lodxprnt9.i

brad-advantzware reviewed (changes requested):
"Customer Name" not changed to "Name"
Use of Global Variables is unacceptable.  There appears to be a temp-table that you use to populate data on these xprint loadtags.  Why would we not add fields to the temp-table and add the "broker" logic to the core program that populates the temp-table?  Don't overwrite existing fields, just use new fields and have the conditional logic to use those fields, in the lodxprint9.i program.

sewa67 committed:
Ticket - 51950 On our Load Tags, we changed all the different shipping addresses to "Broker" , but Atlantic Box info is still
Files - oerep/r-loadtg.w,oerep/r-loadtg.i
oerep/LoadtagProcs.p
oe/rep/lodxprnt9.i
